cornflow-dags/DAG/rostering/README.rst

```diff
@@ -18,13 +18,18 @@ This decision is subject to:
 - RQ06: employees at least work the minimum amount of hours every day.
 - RQ07: employees rest the minimum amount of hours between working days.
 - RQ08: at least one manager is at the work center at all times.
+- RQ09: the demand for each skill should be covered
```

Comment:
Even though the holidays are not present as a constraint in the model, they are requirements of the problem and solution, so they should appear here.